jsapp/js/api.endpoints.ts

```diff
@@ -2,6 +2,8 @@ export const endpoints = {
   PRODUCTS_URL: '/api/v2/stripe/products/',
   SUBSCRIPTION_URL: '/api/v2/stripe/subscriptions/',
   ORGANIZATION_URL: '/api/v2/organizations/',
-  CHECKOUT_URL: '/api/v2/stripe/checkout-link', /** Expected parameters: price_id and organization_uid **/
-  PORTAL_URL: '/api/v2/stripe/customer-portal', /** Expected parameter: organization_uid  **/
+  CHECKOUT_URL:
+    '/api/v2/stripe/checkout-link' /** Expected parameters: price_id and organization_uid **/,
+  PORTAL_URL:
+    '/api/v2/stripe/customer-portal' /** Expected parameter: organization_uid  **/,
```

Comment:
Move these comments to the line above, instead of endline. This way it will still work fine with editors and it would look cleaner :)